src/components/Pf/PfInfoCard.tsx

```diff
@@ -1,24 +1,36 @@
 import * as React from 'react';
 import { Icon } from 'patternfly-react';
+import { IstioLogo } from '../../config';
 
 interface CardProps {
   iconType?: string;
   iconName?: string;
   title?: string;
   items?: any;
+  istio?: boolean;
 }
 
 class PfInfoCard extends React.Component<CardProps> {
   constructor(props: CardProps) {
     super(props);
   }
 
+  istoIcon() {
+    let istioElement;
+    if (this.props.istio) {
+      istioElement = <img className="IstioLogo" src={IstioLogo} alt="Istio sidecar" />;
+    }
+
+    return istioElement;
+  }
+
   render() {
     return (
       <div className="card-pf">
         <div className="card-pf-heading">
           <h2 className="card-pf-title">
             <Icon type={this.props.iconType} name={this.props.iconName} style={{ marginRight: '10px' }} />
+            {this.istoIcon()}
             {this.props.title}
```

Comment:
Not Blocker but why not { this.props.istio && (<img className="IstioLogo" src={IstioLogo} alt="Istio sidecar" />) }

Reply:
Yeah, probably code could be refactor in that way.
But as @xeviknal was the original author and it's more a style side, I let you comment it when he is back and in a future iteration can be refined.